qutebrowser/mainwindow/prompt.py

```diff
@@ -616,6 +616,8 @@ def __init__(self, question, parent=None):
         self._init_texts(question)
         self._init_key_label()
 
+        self._expands_user = False
```

Comment:
I wonder if this is a fitting name. Why "expands user"? Wouldn't be `user_expanded` or so be more fitting?